docker/phpox/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM alpine
-
+FROM alpine:3.10
+#
 # Include dist
 ADD dist/ /root/dist/
-
+#
 # Install packages
-RUN apk -U --no-cache add \
+RUN sed -i 's/dl-cdn/dl-2/g' /etc/apk/repositories && \
```

Comment:
just wondering, why you prefer dl-2 ?

Reply:
At around 14:00 UTC building is so slow from `dl-cdn` that only switching to `dl-2` works reasonably fast :bowtie: